### PR TITLE
Fix issue 3 by reviewing the entrypoint script enabling several commands

### PR DESCRIPTION
**Pull request by mathieupotier**
Still in WIP, need to tests for non regression, but in theory it should allow end user to backup and restore an instance.

A updated the entrypoint in order to ask "run" "version" "backup" and "restore"
Added a `/backups` volume in the container

```bash
docker run -d mgodlewski/diacamma:latest usage
Usage:
      entrypoint.sh [<command_name>]
Available commands:
  Action:
      run                     Run Diacamma instance
  Admin:
      backup                  Backup Diacamma instance to a backup file
      restore                 Restore Diacamma instance from a backup file
  Help:
      version                 Show the Diacamma version running
      usage                   This help screen
      usage <command_name>    Help for the given command
```

Tell me if it suits to you.

Regards

### CODE REVIEW

**Comment by mathieupotier** [comment]
I push these changes for your review @mgodlewski , I will test this on a UNIX system to ensure everything is going well, I'll update the readme after these tests.

Cheers

File: diacamma/entrypoint.d/sub_command/restore.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+function run {
+    pushd /var/lucterios2 > /dev/null
+    filename="/backups/${1}"
+    if [ -f ${filename} ]
+    then
+        /bin/bash launch_lucterios.sh archive --name=${CURRENT_ORGANISATION} --file=${filename}
```
Comment (mgodlewski) [inline diacamma/entrypoint.d/sub_command/restore.sh]: Sound like a typo : should be "restore" instead of "archive" ?

**Comment by mgodlewski** [comment]
Great work !
This buch of script simplify access to backup/restore function.
Once the typo fixed, I'll merge this PR.
Thank you for your contribution.

File: diacamma/entrypoint.d/sub_command/restore.sh (hunk 1 above)
Comment (mathieupotier) [reply diacamma/entrypoint.d/sub_command/restore.sh]: you're right, copy paste issue ... that could explain why it didn't work while testing ... PEBKAC issue ... ^_^

**Comment by mathieupotier** [comment]
I'm taking care of the conflicts this evening ... ^_^

**Pull request merged**